Gruntfile.js

```diff
@@ -39,101 +66,122 @@ module.exports = function( grunt ) {
           "unqualified-attributes": false,
           "vendor-prefix": false,
           "zero-units": false
+        }
+      }
+    },
+    jshint: {
+      default: {
+        options: {
```

Comment:
I've been burned before with JSHint and not setting the things you want.  It is very, very lax now, and you have to turn things on that you want checked.  I think we should add to this list and enable things.